cf-serverd/server_tls.c

```diff
@@ -1078,6 +1078,29 @@ bool BusyWithNewProtocol(EvalContext *ctx, ServerConnectionState *conn)
 
         break;
     }
+    case PROTOCOL_COMMAND_COOKIE:
+    {
+        if (ConnectionInfoProtocolVersion(conn->conn_info) < CF_PROTOCOL_COOKIE)
+        {
+            goto protocol_error;
+        }
+        if (acl_CheckExact(query_acl, "delta",
+                           conn->ipaddr, conn->revdns,
+                           KeyPrintableHash(ConnectionInfoKey(conn->conn_info)))
+            == false)
```

Comment:
I'd add a variable with some descriptive name to store the result of `acl_CheckExact` and then used that in the `if` (with `!` instead of `== false`).

Reply:
Added.